calcom/cal.com
# feat: OOO webhook and zapier

## What does this PR do?



- Fixes #15430
- Fixes CAL-3933


Loom Video: https://www.loom.com/share/b1a4d42ec78944d49248501e2d805c56


## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



- Are there environment variables that should be set? set environment time to UTC
- What are the minimal test data to have? Zapier developer account & app
- What is expected (happy path) to have (input and output)? Create a ooo entry and expect webhooks/zapier triggers
- Any other important info that could help to test that PR For testing webhooks https://webhook.site/

## Checklist


/claim #15430
